### PR TITLE
remove spam_checks_enabled flag

### PR DESCRIPTION
**Pull request by peterbe**
slowly, they go. 
By the way, this was the last waffle flag that still appears in prod for anonymous users:

```
▶ curl https://developer.mozilla.org/api/v1/whoami | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   104  100   104    0     0    318      0 --:--:-- --:--:-- --:--:--   318
{
  "waffle": {
    "flags": {
      "spam_checks_enabled": true
    },
    "switches": {
      "welcome_email": true
    },
    "samples": {}
  }
}
```

### CODE REVIEW

File: kuma/wiki/migrations/0010_auto_20210107_1338.py [hunk 1]
```diff
@@ -14,4 +14,4 @@ class Migration(migrations.Migration):
         ("wiki", "0009_auto_20210105_0406"),
     ]
 
-    operations = []
+    operations = [migrations.RunPython(forward)]
```
Comment (peterbe) [inline kuma/wiki/migrations/0010_auto_20210107_1338.py:17]: Oops!

File: kuma/wiki/migrations/0011_auto_20210108_0416.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.16 on 2021-01-08 04:16
+
+from django.db import migrations
+
+
+def forward(apps, schema_editor):
+    Flag = apps.get_model("waffle", "Flag")
+    Flag.objects.filter(
+        name__in=["kumaediting", "page_move", "section_edit", "spam_checks_enabled"]
```
Comment (peterbe) [inline kuma/wiki/migrations/0011_auto_20210108_0416.py:9]: This should have been done in `0010_auto_20210107_1338.py` but I failed to actually call the `forward` function.

**Comment by peterbe** [comment]
This time, I didn't even bother trying to fix tests. I just deleted a bunch of them. They need to be deleted anyway.

File: kuma/wiki/migrations/0010_auto_20210107_1338.py (hunk 1 above)
Comment (escattone) [reply kuma/wiki/migrations/0010_auto_20210107_1338.py:17]: Sorry, I missed that in my review.

**Review by escattone — APPROVED** [review]
Looks/works grea, thanks @peterbe !

**Pull request merged**